ts-rs/src/export.rs

```diff
@@ -249,3 +256,17 @@ where
         Some(comps.iter().map(|c| c.as_os_str()).collect())
     }
 }
+
+/// Returns an unindented docstring that has a newline at the end if it has content.
+fn format_docs(docs: &str) -> String {
+    match docs.is_empty() {
+        true => "".to_string(),
+        false => {
+            let lines = docs
+                .lines()
```

Comment:
Interestingly, changing docs from `Vec<String>` to `&str` only changed this line (`.iter()` to `.lines()`) in `format_docs`